### ISSUE TITLE
Pulse should install to %localappdata% instead of %appdata%

### ISSUE THREAD

**Issue by fpqc** [aseering/wsl_gui_autoinstall#1]
%appdata% in Windows Multi-User/Domain environment is the roaming appdata that gets moved between PCs.  I think %localappdata% (which is also where lxss lives) makes more sense!

**Comment by aseering** [comment]
Ah, thanks!  I wasn't sure what the difference was; I don't have multiple domain-joined machines so I didn't see that.  Will make that change.

**Comment by aseering** [comment]
Actually:  `%LocalAppData%\Microsoft\Windows\Start Menu\Programs\Startup` doesn't exist on my system.  And if I create it, files there are not opened at login.

The Internets seem to say that my entire design is Wrong(tm); that I should be putting the program in `Program Files (x86)` and the the auto-launch script in the registry.  I don't know off the top of my head how to do the appropriate registry magic, though; also, I kind of want this script to work without admin privileges, and my assumption (maybe misguided / from Linux-land?) would be that installing in a global Program Files could require elevated permissions?

**Comment by fpqc** [comment]
%localappdata% is where I was saying you should install the pulse executable.  It's not so important though.  Program Files is like /bin and can only be written to with Windows admin permissions.

**Comment by aseering** [comment]
Well, if I put the pulse executable in %localappdata% and the launcher for the pulse executable in %appdata%, I guess that means some machine that doesn't have a copy of pulse might try to run it?

**Comment by fpqc** [comment]
Yeah, that's why I was like "yeah, I guess it doesn't really matter".

The real answer probably is for MS to move WSL into roaming instead of local, so you can load up your WSL profile on different networked machines.
